src/flow.h

```diff
@@ -329,6 +329,9 @@ typedef struct Flow_
     /** flow queue id, used with autofp */
     SC_ATOMIC_DECLARE(int16_t, autofp_tmqh_flow_qid);
 
+    /** flow hash - the flow hash before hash table size mod. */
+    uint32_t flow_hash;
```

Comment:
Since this is static after initialization, I think it would be better to move it up in the struct a bit. Comments denote a 'header' part which contains all fields that don't change after flow creation.